library/src/main/java/com/chuckerteam/chucker/internal/support/TransactionDiffCallback.kt

```diff
@@ -0,0 +1,14 @@
+package com.chuckerteam.chucker.internal.support
+
+import androidx.recyclerview.widget.DiffUtil
+import com.chuckerteam.chucker.internal.data.entity.HttpTransactionTuple
+
+internal object TransactionDiffCallback : DiffUtil.ItemCallback<HttpTransactionTuple>() {
```

Comment:
We might consider overriding `getChangePayload` and return i.e. `Unit` from it. This will prevent item animation change and remove flicker.

Reply:
You mean that we could use onBind with payload in adapter to avoid blinking the whole item, but update only changed fields?

Reply:
Yes. :)

Reply:
Got it. Interesting trick. Checked it - with light theme the flicker is less visible for me, but with dark theme your suggestion looks better 👍🏻